Camera.MAUI/Platforms/Android/MauiCameraView.cs

```diff
@@ -212,22 +217,8 @@ internal async Task<CameraResult> StartRecordingAsync(string file, Microsoft.Mau
                             maxVideoSize = new((int)Resolution.Width, (int)Resolution.Height);
                         mediaRecorder.SetVideoSize(maxVideoSize.Width, maxVideoSize.Height);
 
-                        IWindowManager windowManager = context.GetSystemService(Context.WindowService).JavaCast<IWindowManager>();
-
-                        int rotation = (int)windowManager.DefaultDisplay.Rotation;
-                        int orientation = 0;
-
-                        // Set camera rotation based on front or back camera.
-                        if (cameraView.Camera.Position == CameraPosition.Back)
-                            orientation = ORIENTATIONS.Get(rotation);
-                        else
-                            orientation = ORIENTATIONSFRONT.Get(rotation);
-
-                        mediaRecorder.SetOrientationHint(orientation);
-
-                        Console.WriteLine($"Rotation: {rotation} Orientation: {orientation}");
-
-
+                        mediaRecorder.SetOrientationHint(CheckOrientation());
```

Comment:
There's a similar situation in `GetJpegOrientation()`.  It looks like someone started to check it, but never did anything with `sensorOrientation` there?  I don't know if that is an issue or not...